proto/celestia/signal/v1/query.proto

```diff
@@ -13,6 +14,12 @@ service Query {
       returns (QueryVersionTallyResponse) {
     option (google.api.http).get = "/upgrade/v1/tally/{version}";
   }
+
+  // GetUpgrade enables a client to query for upgrade information if an upgrade is pending.
```

Comment:
nit: maybe add a little note that the response will be empty if there is no current upgrade pending

Reply:
01ec9f3d0e8089483f7e5d887b121622e5088719